provider/internal/user/resource_test.go

```diff
@@ -9,7 +9,7 @@ import (
 
 type testUserResourceType struct {
 	Name        string
-	ApiOnlyUser bool
+	ApiOnlyUser string
```

Comment:
Please don't change this to a string here; couldn't you just convert `acctest.Env.UserResourceIsApiOnly(),` to a boolean? Otherwise it's not very clear.

Reply:
Sure will do that, I was thinking it is going to be converted into string again very soon anyway because terraform requires it to be string when passing it for checks.